bin/preview.sh

```diff
@@ -66,20 +66,23 @@ if [[ -z "$BATCAT" ]]; then
   fi
 fi
 
-if [ -z "$FZF_PREVIEW_COMMAND" ] && [ "${BATCAT:+x}" ]; then
+if [ -z "$FZF_PREVIEW_COMMAND" ] && [ "${BATCAT:+x}" ] && [[ ! -d "$FILE" ]] ; then
   ${BATCAT} --style="${BAT_STYLE:-numbers}" --color=always --pager=never \
       --highlight-line=$CENTER -- "$FILE"
   exit $?
 fi
 
 FILE_LENGTH=${#FILE}
 MIME=$(file --dereference --mime -- "$FILE")
-if [[ "${MIME:FILE_LENGTH}" =~ binary ]]; then
+if [[ "${MIME:FILE_LENGTH}" =~ binary ]] && [[ ! -d "$FILE" ]]; then
   echo "$MIME"
   exit 0
 fi
 
 DEFAULT_COMMAND="highlight -O ansi -l {} || coderay {} || rougify {} || cat {}"
+if [[ -d "$FILE" ]]; then
+  DEFAULT_COMMAND="tree -C -L2 {} || ls -l --color=always {}"
```

Comment:
Are we doing this in case `tree` is not available on the system? If that's the case, we might want to suppress the error message when it's non-existent.

Reply:
Nevermind, it's already done.